src/card.css

```diff
@@ -0,0 +1,98 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card__product-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product-name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin-bottom: 0;
+  margin-top: 40px;
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card___stars-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__reviews {
+  text-align: right;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  padding-top: 24px;
+  justify-content: space-between;
+}
+
+.card__value-text {
+  text-align: left;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-number {
+  text-align: right;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  box-sizing: border-box;
+  margin-top: 16px;
+  background: var(--primary-color);
+  border-radius: 5px;
+  bottom: 17px;
+  text-transform: uppercase;
+  text-align: center;
+  border: 1px solid;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  padding: 12px 0;
+  color: white;
+  display: block;
```

Comment:
default value

Reply:
By saying it's default I meant you can remove this aswell. It's the last thing :)

Reply:
I don't know which of these four lines is default, but removing any of these destroys button completly. Font size and padding give the 'shape' of the button, color changes default black, and becouse my button is not in div, but its modified 'a', default display is not block either.

Reply:
You're right, mb